### DISCUSSION TITLE
v0.6.2

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#1518] [Versions & Releases]
## What's Changed
* Fix multilingual recipe by @Edresson in https://github.com/coqui-ai/TTS/pull/1354
* Fix recipes as to the recent API changes.  by @erogol in https://github.com/coqui-ai/TTS/pull/1367
* Add docsqa to docs website by @nomagick in https://github.com/coqui-ai/TTS/pull/1363
* REBASED: Add support for the speaker encoder training using torch spectrograms  by @Edresson in https://github.com/coqui-ai/TTS/pull/1348
* Add alphas to control language and speaker balancer by @Edresson in https://github.com/coqui-ai/TTS/pull/1216
* Add Voice conversion inference support by @Edresson in https://github.com/coqui-ai/TTS/pull/1337
* Update issue template by @erogol in https://github.com/coqui-ai/TTS/pull/1370
* Open bible dataset formatter by @Edresson in https://github.com/coqui-ai/TTS/pull/1365
* REBASED: Transform Speaker Encoder in a Generic Encoder and Implement Emotion Encoder training support  by @Edresson in https://github.com/coqui-ai/TTS/pull/1349
* Fix typo workflow text by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1403
* Add CITATION.cff by @erogol in https://github.com/coqui-ai/TTS/pull/1404
* Fix default phonemizer for ja and zh by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1399
* Make Style by @erogol in https://github.com/coqui-ai/TTS/pull/1405
* Fix #1380 by @erogol in https://github.com/coqui-ai/TTS/pull/1409
* Hinge Gruut version to 2.2.3 by @erogol in https://github.com/coqui-ai/TTS/pull/1419
* Update CheckSpectrograms notebook by @erogol in https://github.com/coqui-ai/TTS/pull/1418
* Fix #1423 by @Edresson in https://github.com/coqui-ai/TTS/pull/1424
* Update model file extension by @erogol in https://github.com/coqui-ai/TTS/pull/1422
* Fix model manager  by @erogol in https://github.com/coqui-ai/TTS/pull/1436
* Add formatting tests by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1437
* Update base model wrt 👟 by @erogol in https://github.com/coqui-ai/TTS/pull/1406
* Replace webrtcvad by silero-vad by @Edresson in https://github.com/coqui-ai/TTS/pull/1431
* Bug fix in freeze encoder by @Edresson in https://github.com/coqui-ai/TTS/pull/1391
* Enforce phonemizer definition for synthesis by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1441
* Fix G2P backend of the released models by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1461
* Add EmbeddingManager and BaseIDManager by @Edresson in https://github.com/coqui-ai/TTS/pull/1374
* Update requirements coqui_trainer -> trainer by @erogol in https://github.com/coqui-ai/TTS/pull/1478
* Update CONTRIBUTING.md, fix header by @Jackiexiao in https://github.com/coqui-ai/TTS/pull/1463
* Add African models by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1511
* Print Model's license when downloading   by @erogol in https://github.com/coqui-ai/TTS/pull/1512
* Improve docsQA default questions by @nomagick in https://github.com/coqui-ai/TTS/pull/1411
* patch print license by @WeberJulian in https://github.com/coqui-ai/TTS/pull/1514
* v0.6.2 by @erogol in https://github.com/coqui-ai/TTS/pull/1353

## New Contributors
* @nomagick made their first contribution in https://github.com/coqui-ai/TTS/pull/1363
* @Jackiexiao made their first contribution in https://github.com/coqui-ai/TTS/pull/1463

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.6.1...v0.6.2

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.6.2'>v0.6.2</a>.</em>

**Comment by godspirit00** [comment]
Thanks for the great work!
Is there a tutorial for the voice conversion inference?
I tried ```tts --model_name "tts_models/en/vctk/vits" --reference_wav "J:/1646276861.wav" --speaker_idx p250 --out_path "D:/1.wav"```
but got 
```
Traceback (most recent call last):
  File "C:\Users\xxx\Envs\coqui\Scripts\tts-script.py", line 33, in <module>
    sys.exit(load_entry_point('TTS', 'console_scripts', 'tts')())
  File "d:\github\tts\TTS\bin\synthesize.py", line 305, in main
    wav = synthesizer.tts(
  File "d:\github\tts\TTS\utils\synthesizer.py", line 336, in tts
    outputs = transfer_voice(
  File "d:\github\tts\TTS\tts\utils\synthesis.py", line 273, in transfer_voice
    model_outputs = _func(reference_wav, speaker_id, d_vector, reference_speaker_id, reference_d_vector)
  File "C:\Users\xxx\Envs\coqui\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "d:\github\tts\TTS\tts\models\vits.py", line 1022, in inference_voice_conversion
    wav, _, _ = self.voice_conversion(y, y_lengths, speaker_cond_src, speaker_cond_tgt)
  File "d:\github\tts\TTS\tts\models\vits.py", line 1039, in voice_conversion
    g_src = self.emb_g(speaker_cond_src).unsqueeze(-1)
  File "C:\Users\xxx\Envs\coqui\lib\site-packages\torch\nn\modules\module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "C:\Users\xxx\Envs\coqui\lib\site-packages\torch\nn\modules\sparse.py", line 158, in forward
    return F.embedding(
  File "C:\Users\xxx\Envs\coqui\lib\site-packages\torch\nn\functional.py", line 2183, in embedding
    return torch.embedding(weight, input, padding_idx, scale_grad_by_freq, sparse)
TypeError: embedding(): argument 'indices' (position 2) must be Tensor, not NoneType
```
Not sure if I got it right. Any help please? Thank you!

**Reply by erogol** [reply]
That model does not use reference audio. It just takes a speaker ID. Reference audio is only necessary with YourTTS. 

But I guess we need better messaging to inform the user.
